commit 34df9d81d15b596ea6f3f3f2e15bd0ec55323a19
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Sat Nov 17 17:57:59 2012 +0200

    Fix double-free on %caps in spec (RhBug:877512)
    
    - Fixes regression caused by dumb refactoring mistake in commit
      807ba93a6cf168410e3801347898949f356fcd6c

diff --git a/build/files.c b/build/files.c
index d299cb9..10d7fff 100644
--- a/build/files.c
+++ b/build/files.c
@@ -216,7 +216,6 @@ static void FileEntryFree(FileEntry entry)
 {
     freeAttrRec(&(entry->ar));
     argvFree(entry->langs);
-    free(entry->caps);
     memset(entry, 0, sizeof(*entry));
 }
 
